recipe/meta.yaml

```diff
@@ -25,9 +25,11 @@ requirements:
 test:
   imports:
     - coverage
-
+  requires:
+    - pip
   commands:
     - coverage --help
+    - pip check
 
 about:
   home: https://coverage.readthedocs.io
```

Comment:
Please add `license_family: Apache`

And  the license should be: 
`license: Apache-2.0`

Reply:
The `license` and `license_family` were updated:

https://github.com/AnacondaRecipes/coverage-feedstock/blob/update_to_6.1.1/recipe/meta.yaml#L39&L40